zebra-rpc/src/methods.rs

```diff
@@ -48,15 +52,49 @@ pub trait Rpc {
     ///       note any other lightwalletd changes
     #[rpc(name = "getblockchaininfo")]
     fn get_blockchain_info(&self) -> Result<GetBlockChainInfo>;
+
+    /// getblock
+    ///
+    /// Returns requested block by height, encoded as hex.
+    ///
+    /// zcashd reference: <https://zcash.github.io/rpc/getblock.html>
+    ///
+    /// Result:
+    /// {
+    ///      "data": String, // The block encoded as hex
+    /// }
+    ///
+    /// Note: We only expose the `data` field lightwalletd uses the non-verbose
+    /// mode for all getblock calls: <https://github.com/zcash/lightwalletd/blob/v0.4.9/common/common.go#L232>
+    #[rpc(name = "getblock")]
+    fn get_block(&self, height: Height) -> BoxFuture<Result<GetBlock>>;
```

Comment:
```
$ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0", "method": "getblock", "params": [4, 0], "id":123 }' 127.0.0.1:8232 | jq
{
  "jsonrpc": "2.0",
  "error": {
    "code": -32602,
    "message": "Invalid params: invalid length 2, expected fewer elements in array."
  },
  "id": 123
}
$ 
```
Ill fix.

Reply:
Added parameter in https://github.com/ZcashFoundation/zebra/pull/3707/commits/efe12fac0d9b31e76fb8719fa7a9ede56aac1bd9.

Now this will work:

```
$ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0", "method": "getblock", "params": [4, 0], "id":123 }' 127.0.0.1:8232 | jq
{
  "jsonrpc": "2.0",
  "result": {
    "data": "040000007fc6ea65cae94cd84309ed9e0f35a ..."
  },
  "id": 123
}
$
```

However calling without the right number of arguments  will not:

```
$ curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0", "method": "getblock", "params": [4], "id":123 }' 127.0.0.1:8232 | jq
{
  "jsonrpc": "2.0",
  "error": {
    "code": -32602,
    "message": "Invalid params: invalid length 1, expected a tuple of size 2."
  },
  "id": 123
}
$
```